roles/scenario1/templates/create_env.sh.j2

```diff
@@ -36,8 +36,14 @@ $OC_COMMAND openstack flavor show m1.small > /dev/null 2> /dev/null || $OC_COMMA
 
 $OC_COMMAND openstack security group create scenario-{{ scenario }}-sg
 
-$OC_COMMAND openstack keypair create scenario-{{ scenario }}-key > {{ ssh_key_file }}
-chmod 0600 {{ ssh_key_file }}
+if type ssh-keygen > /dev/null 2>&1; then
+    yes | ssh-keygen -t ecdsa -f {{ ssh_key_file }} -N "" > /dev/null
```

Comment:
Doesn't CLI generate elyptic curve (Ed25519) keypair? https://github.com/openstack/python-openstackclient/commit/2454636386d443473dedff1f07f8623108e87298

Reply:
OK, I see that nova (pre 2.92 microversion) indeed generates RSA keys. Is the patch above not in Antelope? Maybe we can add a TODO to revert to bare `keypair create` once client is updated.

Reply:
done